client/extensions/woocommerce/app/settings/email/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import MailChimp from './mailchimp';
+import ActionHeader from 'woocommerce/components/action-header';
+import SettingsNavigation from '../navigation';
+import { getLink } from 'woocommerce/lib/nav-utils';
+
+const Email = ( { site, siteId, translate, className } ) => {
```

Comment:
Would you mind naming this `SettingsEmail` instead - like we did for Payments and Taxes (but unfortunately did not do for Shipping)

This should make it easier to import without collision for such a common term as Email